posts/2023/10/hpy_meetup.md

```diff
@@ -0,0 +1,135 @@
+<!--
+.. title: HPy meetup and CPython core dev sprint in Brno (Oct 7-10, 2023)
+.. slug: hpy-meetup-2023
+.. date: 2023-10-10 10:00:00 UTC
+.. author: fangerer
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+We just ended our rather informal HPy meetup in Brno (Czech Republic) which
+started on Saturday (October 7th) and lasted until Tuesday (October 10th).
+
+The main topics of the HPy meetup were:
+
+- The CPython Core Dev Sprint
+- Cython/HPy
+
+<!--TEASER_END-->
+
+# The CPython Core Dev Sprint
+
+Petr Viktorin, one of the most active CPython core developers, invited us to
+partially attend the CPython core developer sprint. As a part of the sprint, he
+organized a C API Summit where we were invited and happily attended this event.
+
+Besides of lots of discussions, there were also very interesting presentations
+about the C API:
+
+## 1. C API Work Group Summary
+
+Irit presented the current status of the [C API Work Group
+GitHub](https://github.com/capi-workgroup) organisation. The idea of this GitHub
```

Comment:
She just published her slides at: https://github.com/iritkatriel/talks/blob/main/2023_Sprint_Brno_C_API.pdf

You can add a link ;-)